sdk/identity/azure-identity/src/main/java/com/azure/identity/DefaultAzureCredential.java

```diff
@@ -37,6 +38,7 @@
         super(new ArrayDeque<>(Arrays.asList(new EnvironmentCredential(identityClientOptions),
             new ManagedIdentityCredential(null, identityClientOptions),
             new SharedTokenCacheCredential(null, "04b07795-8ddb-461a-bbee-02f9e1bf7b46",
-                identityClientOptions))));
+                identityClientOptions),
+            new AzureCliCredential(identityClientOptions))));
```

Comment:
Is this correct order for AzureCliCredential @jianghaolu  ?

Reply:
Yes this would be the correct ordering.